doc/project/module.md

```diff
@@ -170,33 +228,84 @@ There are currently two module split styles: `FewestModules` and `SmallestModule
 
 #### `FewestModules`
 
-Create as few modules as possible without including unnecessary code.
+Create as few modules as possible
+* while respecting dynamic import boundaries and
+* without including unnecessary code.
+
 This is the default.
 
-In the example above, this would generate:
+In the entry points example above, this would generate:
 
 * `a.js`: public module, containing `AppA` and the export of `start`.
 * `b.js`: public module, containing `AppB`, `mutable.Set`, the export of `start` and the call to `AppB.main`
 * `a-b.js`: internal module, Scala.js core and the implementation of `println`.
 
 This also works for more than two public modules, creating intermediate shared (internal) modules as necessary.
 
+The dynamic import example above already assumes this module split style so a module listing is omitted.
+
 #### `SmallestModules`
 
 Create modules that are as small as possible.
-The smallest unit of splitting is a Scala class.
+The smallest unit of splitting is a Scala class (see [Splitting Granularity](#Splitting Granularity) below for more).
```

Comment:
Is this the correct way to link to an anchor?

Reply:
Yes, except that the titles are converted to something `#splitting-granularity`. I don't try to predict it, and instead generate then inspect the page with a browser's inspection tools...